include/logging/log_rpc.h

```diff
@@ -20,6 +20,25 @@
 extern "C" {
 #endif
 
+/**
+ * nRF RPC logging level.
+ */
+enum log_rpc_level {
+	LOG_RPC_LEVEL_NONE = 0,
+	LOG_RPC_LEVEL_ERR,
+	LOG_RPC_LEVEL_WRN,
+	LOG_RPC_LEVEL_INF,
+	LOG_RPC_LEVEL_DBG,
+};
+
+/**
+ * Sets the logging stream verbosity level.
+ *
+ * Configures the remote device to only stream log messages whose level is less
+ * than or equal to the specified level.
+ */
+int log_rpc_set_stream_level(enum log_rpc_level level);
```

Comment:
Public API should have sufficient description of incoming parameters and return value with Doxygen tags.

Reply:
Right, done.